test/lib/middleware/bloc_test.dart

```diff
@@ -104,15 +115,8 @@ void main() {
 
     test('BLoC Strategy Test', () async {
       await mockFileStructureStrategy.setUpDirectories();
-      var tempNode = await bLoCMiddleware.applyMiddleware(tree);
-      expect(tempNode, isNull);
-
-      var verification =
-          verify(mockFileStructureStrategy.commandCreated(captureAny));
-
-      expect(verification.captured[0].fileName, contains('state'));
```

Comment:
Is there a reason for removing these? I think we should still be checking that the commands are successfully called 🤔